### PR TITLE
EJ2-Documentation(849693) - UG documentation for the Row drag and drop

### PR DESCRIPTION
**Pull request by Gayathri4135**
Modified the drag and drop in row for both javascript and typescript

### CODE REVIEW

File: ej2-javascript/code-snippet/grid/draganddrop-event-cs1/index.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    rowDrag: function (args: RowDragEventArgs) {
+      (document.getElementById('message') as HTMLElement).innerText = `rowDrag event triggered`;
+      args.rows.forEach((row) => {
+        row.classList.add('drag-limit');
+      });
+    },
+    rowDrop: function (args: RowDragEventArgs) {
+      (document.getElementById('message') as HTMLElement).innerText = `rowDrop event triggered`;
+      let value = [];
+      for (let r = 0; r < args.rows.length; r++) {
```
Comment (BalajiSGrid) [inline ej2-javascript/code-snippet/grid/draganddrop-event-cs1/index.ts:36]: replace the "r" to "index"

File: ej2-javascript/grid/row/row-drag-and-drop.md [hunk 2]
```diff
@@ -9,79 +9,88 @@ documentation: ug
...
 
-In the below example, the selected grid row is dragged and dropped in to the TreeView component by using [`rowDrop`](../../api/grid/#rowdrop) event. Once the row is dropped into the TreeView component, we have removed the corresponding grid row from grid and its data inserted in custom component. To know more information about the drag and drop events check [`here`](https://ej2.syncfusion.com/documentation/grid/row/?no-cache=1#drag-and-drop-events).
+In the below example, the selected grid row is dragged and dropped in to the TreeView component by using [rowDrop](../../api/grid/#rowdrop) event. Once the row is dropped into the TreeView component, we have removed the corresponding grid row from grid and its data inserted in custom component.
```
Comment (BalajiSGrid) [inline ej2-javascript/grid/row/row-drag-and-drop.md:93]: We have used the custom component as Treegrid, not Treeview so please replace on this line

File: ej2-javascript/code-snippet/grid/draganddrop-event-cs1/index.ts (hunk 1 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/draganddrop-event-cs1/index.ts]: Changed the "r" to index

File: ej2-javascript/grid/row/row-drag-and-drop.md (hunk 2 above)
Comment (Gayathri4135) [reply ej2-javascript/grid/row/row-drag-and-drop.md]: changed treeView to treeGrid

**Comment by BalajiSGrid** [comment]
The demo and technical review have been completed.

**Comment by MaithiliyK** [comment]
Making PR is new hotfix branch and the changes merged too https://github.com/syncfusion-content/ej2-grid-docs/pull/131 so i closed this PR

**Pull request closed without merging**